src/Screens/App/Components/Todo/styles.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingHorizontal: 10,
+    paddingVertical: 8,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    height: 50,
+    alignItems: 'center'
+  },
+  todoText: {
+    fontWeight: 'bold'
+  },
+  checkbox: {
+    borderColor: '#000',
+    borderWidth: 2,
+    borderRadius: 5,
+    height: 20,
+    width: 20,
```

Comment:
I'd extract this `20` to a `const` within this file, since this will always behave as a square